src/example/main.js

```diff
@@ -283,6 +283,8 @@ const examples = (
 // Cannot render to body anymore: react is throwing warnings.
```

Comment:
You could make it more clear in the comments that this is needed to support React 14 (I assume that is the reason we do this).